addons/sys_core/fnc_inZeus.sqf

```diff
@@ -0,0 +1,14 @@
+/*
+ * Author: SynixeBrett
+ * Returns true if the local player is in the Zeus interface
+ *
+ * Return Value:
+ * In Zeus Interface <BOOL>
+ *
+ * Example:
+ * _inZeus = call acre_sys_core_fnc_inZeus
+ *
+ * Public: No
+ */
+
+!isNull(findDisplay 312)
```

Comment:
space between `isNull` and `findDisplay`. Additionally the 312 should be a  `#define`